tests/test_morphsqueeze.py

```diff
@@ -0,0 +1,74 @@
+"""
```

Comment:
we don't need docstrings in test functions, so please delete this, (but we do like comments on each of the tests that capture the intended behavior of each test.